pkg/trafficpolicy/trafficpolicy.go

```diff
@@ -116,22 +95,15 @@ func (out *OutboundTrafficPolicy) AddRoute(httpRouteMatch HTTPRouteMatch, retryP
 // 2. when a policy having its hostnames from a host header needs to be merged with other inbound traffic policies
 // in either of these cases the will be only a single hostname and there is a possibility that this hostname is part of an existing traffic policy
 // hence the rules need to be merged
-func MergeInboundPolicies(allowPartialHostnamesMatch bool, original []*InboundTrafficPolicy, latest ...*InboundTrafficPolicy) []*InboundTrafficPolicy {
```

Comment:
Why is this being changed?

Reply:
it's only ever used in one context, with allowPartialHostnamesMatch set to true